### PR TITLE
[DS-331]: [Pyxis React] Loader component

### PR DESCRIPTION
**Pull request by emanuelemacri**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-331

## 🚀 New Features
- Implementation of Loader component in pyxis-react.
## 📝 Docs
- The documentation of the Loader has been updated.
- Remove `defaultValue` in stories (will be removed in version 7.0 of storybook)

### CODE REVIEW

File: packages/pyxis-react/.eslintrc.js [hunk 1]
```diff
@@ -5,12 +5,13 @@ module.exports = {
     es2021: true,
   },
   ignorePatterns: [
+    '**/Icons/*',
     'dist/',
     'node_modules/',
+    'src/components/Loader/Car.tsx',
```
Comment (alessiosantangelo) [inline packages/pyxis-react/.eslintrc.js:11]: Why we should exclude this file? Which kind of issues does it have?

File: packages/pyxis-react/src/components/Loader/Loader.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+const getClasses = (
+  size: LoaderSize,
+  type: LoaderType,
+  alt: boolean,
+  className?: string,
+): string => classNames(
+  'loader',
+  {
+    'loader--small': size === 'small' && type === 'spinner',
```
Comment (alessiosantangelo) [inline packages/pyxis-react/src/components/Loader/Loader.tsx:13]: If I am right, passing only `size=small` without having the correct `type=spinner` doesn't change anything. 
We should ensure better this cannot be done.

**Review by alessiosantangelo — CHANGES_REQUESTED** [review]

File: packages/pyxis-react/src/components/Loader/Car.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+
+const Car: FC = () => (
+  <svg aria-hidden="true" height="50" viewBox="0 0 375 125">
+    <g className="loader__car__items-wrapper">
+      <path className="loader__car__wheel" d="M184.9,106.3c-12.2,0-21.8-9.6-21.8-21.8s9.6-21.8,21.8-21.8c12.2,0,21.8,9.6,21.8,21.8 S197.1,106.3,184.9,106.3z M184.9,71.2c-7.4,0-12.8,5.9-12.8,12.8c0,7.4,5.9,12.8,12.8,12.8c7.4,0,12.8-5.9,12.8-12.8 C198.2,77.1,192.4,71.2,184.9,71.2z M281.2,106.3c-12.2,0-21.8-9.6-21.8-21.8s9.6-21.8,21.8-21.8c12.2,0,21.8,9.6,21.8,21.8 S292.9,106.3,281.2,106.3z M281.2,71.2c-7.4,0-12.8,5.9-12.8,12.8c0,7.4,5.9,12.8,12.8,12.8c7.4,0,12.8-5.9,12.8-12.8 S288.1,71.2,281.2,71.2z" />
```
Comment (brianvanburken) [inline packages/pyxis-react/src/components/Loader/Car.tsx:6]: Do we optimize/pre-optimize using SVGO?

File: packages/pyxis-react/src/stories/components/Loader/Overview.tsx [hunk 4]
```diff
@@ -151,9 +145,9 @@ const tableClassBody: TableRow[] = [
 
 const Overview: FC = () => (
   <>
-    <OverviewTemplate category="Component" description={overviewDescription} isMain title="Loader 🚧">
+    <OverviewTemplate category="Component" description={overviewDescription} isMain title="Loader">
       <Canvas>
-        <Story id="components-display-loader-🚧-all-stories--spinner-loader" />
+        <Story id="components-display-loader-all-stories--spinner-loader" />
```
Comment (brianvanburken) [inline packages/pyxis-react/src/stories/components/Loader/Overview.tsx:150]: One thing I noticed is when you change names, by removing the emoji for example, the old storybook snapshots are still there. They should be removed manually

File: packages/pyxis-react/src/components/Loader/Car.tsx (hunk 3 above)
Comment (emanuelemacri) [reply packages/pyxis-react/src/components/Loader/Car.tsx:6]: Yep, the path is already optimized, but of course in this case we need the `className` for adding styles and animation

File: packages/pyxis-react/.eslintrc.js (hunk 1 above)
Comment (emanuelemacri) [reply packages/pyxis-react/.eslintrc.js:11]: It's an SVG icon, we have an issue with the `max-len` of eslint, but maybe is better to add a simple comment in the file:
```
/* eslint-disable max-len */
```

**Review by brianvanburken — APPROVED** [review]

**Review by alessiosantangelo — APPROVED** [review]

**Pull request merged**